test/runtests.py

```diff
@@ -454,6 +454,9 @@ def _run_one_test(self, test):
                     args.flags.split() + \
                     flags.split()
 
+        if test.get('custom-config-file') != None:
+            flags = ['-CustomConfigFile:' + test.get('custom-config-file')]
```

Comment:
This overwrites any flags set earlier in this function?

Reply:
This is how the implementation in rl appears to work (see https://github.com/Microsoft/ChakraCore/blob/master/bin/rl/rlrun.cpp#L534), so I attempted to mimic this behavior.